ngx-tools/utilities/src/http-retryer/http-retryer.ts

```diff
@@ -22,7 +22,7 @@ import {
 import { exponentialBackoffDelayCalculator } from './../retry-with-backoff/delay-calculator';
 
 
-export interface RetryWithBackoff {
+export interface HttpRetryWithBackoff {
   retries: number;
   delayCalculator: (attempt: number) => number;
   // TODO: Scheduler is marked as deprecated to stop others from using although it is not technically deprecated from what I can tell. The
```

Comment:
We don't _really_ have a scheduler. The JWT did have to pull it's scheduler out into a separate file to avoid duplicate exports, but we don't really expose any scheduler as an ngx tool utility.

We may formalize a scheduler once we tackle the deprecation `TODO` around the rxjs scheduler.